shared-arguments/src/lib.rs

```diff
@@ -10,6 +11,19 @@ pub struct Arguments {
         default_value = "warn,orderbook=debug,solver=debug"
     )]
     pub log_filter: String,
+
+    /// The Ethereum node URL to connect to.
+    #[structopt(long, env = "NODE_URL")]
```

Comment:
The local host as default would be nice for testing

Reply:
Changed default to `http://localhost:8545`.

Reply:
Not a big deal but personally I find the node_url shouldn't have a default as it's the one thing you should deliberately think about and specify when running this thing (what network am I on).